frontend/packages/kubevirt-plugin/integration-tests-cypress/utils/const/index.ts

```diff
@@ -15,6 +15,7 @@ export const READY_FOR_CUSTOMIZATION = 'Ready for customization';
 export const YAML_VM_NAME = 'vm-example';
 export const STATUS_READY = 'Ready';
 export const TEMPLATE_SUPPORT = 'Support by template provider';
+export const NAD_NAME = 'bridge-network'; // refer to fixture/nad.ts
 
 export enum DEFAULTS_VALUES {
```

Comment:
I know this isn't really a part of this PR, but could you please rename this to 'DEFAULT_VALUES'?

Reply:
Sure, will change the name.